assessments-api-client/src/ApiClient.js

```diff
@@ -43,13 +43,13 @@
    * @alias module:ApiClient
    * @class
    */
-  var exports = function() {
+  var exports = function(basePath) {
```

Comment:
This file is actually auto-generated from the OpenAPI spec, so I think it would be best not to make manual changes to it. I mean, here, in practice, it doesn't really matter since we'll be throwing this away shortly so it's not like there's much of a maintainability concern.

Reply:
That's a good point. I assumed we wouldn't be using auto-generated code for production apps, as it'd create similar "this is code we could change, but don't" concerns. Is that incorrect?